docs/user/autoreduction.rst

```diff
@@ -32,3 +32,38 @@ User can also run the script manually, for instance:
 This call will not publish the results to the live data server,
 but will save report file `EQSANS_89157.html` in the `/tmp` directory.
 User can point the web browser to the generated report file file:///tmp/EQSANS_89157.html.
+
+Input Configuration
+-------------------
+The autoreduction script will automatically look for an input configuration file in three locations,
+in the following order of precedence:
+
+1. Under the `shared` directory of the current IPTS and taking into account the run number.
```

Comment:
In this document, can you please change the single back ticks to double back ticks to format as code? (Or maybe it is it intentional to format them in italics?)